modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/CKEditorBalloonEditorConfigContributor.java

```diff
@@ -48,13 +59,16 @@ public void populateConfigJSONObject(
 
 		jsonObject.put("extraAllowedContent", Boolean.TRUE);
 
+		String balloonToolbarPlugins =
+			"balloontoolbar,floatingspace,stylescombo";
```

Comment:
@ambrinchaudhary just out of curiosity, did this (adding the `stylescombo` plugin) fix the issue you mentioned a few day ago via Slack about the `Styles` button not working properly?

Reply:
Nop, the issue is still there. This plugin is just needed for the styles dropdown to appear... I will look into the issue later

Reply:
I see thanks for the information. I'll have a look too in case I see something.
From what I've seen any "button" with  a dropdown menu (like `BGColor`, `TextColor`) is currently failing to work properly when used in the `balloontoolbar`, it might be a configuration issue or it might be that the plugin isn't prepared for that (I wouldn't be surprised given the quality of the plugin).

Reply:
Yes, about that issue, I opened a bug in CKEditor and for now they just confirmed it https://github.com/ckeditor/ckeditor4/issues/4639

Reply:
Good idea @ambrinchaudhary.
We'll probably have to figure out other options and try to find alternatives if we can't fix the bug ourselves.